kubernetes/mysql-statefullset.yml

```
@@ -0,0 +1,48 @@
+      containers:
+      - name: sql-pod
+        image: mysql:latest
+        ports:
+        - containerPort: 3306
+        environment:
+        - name: MYSQL_ROOT_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: bank-secret
+              key: MYSQL_ROOT_PASSWORD
+        - name:  MYSQL_DATABASE
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)